plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveHadoop.java

```diff
@@ -72,6 +75,17 @@ protected void setupContainer()
                         runCmd));
     }
 
+    @Override
+    protected void startContainer()
+    {
+        super.startContainer();
+        log.info(
+                "Hive container started with addresses for SOCKS proxy: %s; Metastore: %s; HiveServer2: %s",
+                getMappedHdfsSocksProxy(),
```

Comment:
proxy port is unused, right?
maybe we just remove it?

Reply:
Good point. It's not used yet in integration test so yes we might remove it for now.